distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestRegistry.java

```diff
@@ -130,6 +118,26 @@ public void testCswRegistryUpdate() throws Exception {
     }
 
     @Test
+    public void testCswRegistryUpdateFailure() throws Exception {
+        String id = createRegistryEntry("urn:uuid:2014ca7f59ac46f495e32b4a67a51280");
+
+        given().body(Library.getCswRegistryUpdate()
```

Comment:
If you take a look at https://github.com/codice/ddf/blob/master/distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/platform/TestSecurity.java#L923 you can see what we do there to build the message bodies. We're still just replacing strings, but we are replacing known defined keys with generated values as opposed to hardcoding everything.